fixtures/customize/installer-test.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Aren't NMState files YAML? Or does it support both YAML and JSON? (Or is this actually JSON being parsed as YAML?)

Would be good to add a YAML fixture too/instead.